app/views/new_filters/_qualifications_filter.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="govuk-form-group app-filter__group" data-qa="filters__qualifications">
+  <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+    <h3 class="govuk-fieldset__heading">Qualifications</h3>
+  </legend>
+  <div class="govuk-checkboxes govuk-checkboxes--small">
+    <div class="govuk-checkboxes__item">
+      <%=
+        form.check_box(
```

Comment:
There's probably a good reason, but can we use govuk_check_boxes_fieldset here?

Reply:
Find does not currently use `gem 'govuk_design_system_formbuilder'`. I tried implementing and ran into some issues.

From PR description  ... "Note that the new filters are not a good candidate for govuk_form_builder, as validations are not required, plus there are some complications around hydrating filter form values from params"

Specifically I struggled to find a way to 'check' the checkbox via params. I think Apply ran into this issue also and are not using `govuk_check_boxes` currently either (where the checked value needs to  be set)

Reply:
Yea we have to manually retain the values by messing with values in the controller. Like i said probably a good explanation haha